### DISCUSSION TITLE
How do I hide file paths in Search Edit and Export tab

### DISCUSSION THREAD

**Discussion by lsmith212a** [pimcore/pimcore#16219] [Q&A / Support]
Can I hide the file path for related objects in the Search Edit and Export tab? For display on screen and for the csv export?
![image](https://github.com/pimcore/pimcore/assets/57369009/60d2d069-839d-4b4b-a09f-aa1af9623445)

**Comment by dvesh3** [comment]
@lsmith212a you can use "Grid Options" to customize the grid columns and then save a copy. If you want to load the copy every time you open the folder then mark this copy as favorite.
![image](https://github.com/pimcore/pimcore/assets/5137917/8aff1c3f-ca19-4d24-abee-37fe9be35efb)

**Reply by lsmith212a** [reply]
Thank you @dvesh3 . I don't see how to apply a formatter in the Grid Configuration for relationship fields. Am I missing something?

For example, today it's showing Merchandising/Lineups/Cover Lineup/Fabric/B1439.

I just want it to display B1439, the name of the object that is related. Not the whole folder path.

**Reply by dvesh3** [reply]
you can fetch related object field with "ObjectField Getter(Extractor)" and using "key" in the attribute.
